src/app/widgets/exchange/exchange.component.scss

```diff
@@ -0,0 +1,33 @@
+.card {
+  background-color: rgb(51, 19, 19);
+  color: white;
+}
+
+hr {
+  width: 90%;
+  align-self: center;
+}
+
+.currencyPanel {
```

Comment:
snake case or dash case could be used here for class names, also maybe a parent "panel" class could be added, using the scss perks you could then bellow add the children classes " &--currency " and  " &--admin", since some properties of these classes are identical, for example width, padding, border-radius etc.

Reply:
.panel {
  width..
  padding...
  border-radius..
   &--currency {
  unique properties here
}
   &--admin {

}
}